net/tcpreplay/Makefile

```diff
@@ -35,7 +35,7 @@ define Package/tcpreplay/default
   CATEGORY:=Network
   URL:=http://tcpreplay.appneta.com/
   MAINTAINER:=Alexandru Ardelean <ardeleanalex@gmail.com>
-  DEPENDS:=+librt +libpcap +libdnet +USE_MUSL:musl-fts
+  DEPENDS:=+librt +libpcap +libdnet +libbpf +USE_MUSL:musl-fts
```

Comment:
@teleostnacl 

are you sure `libbpf` will be used ?

if i look into 
```
CONFIGURE_ARGS += \
	--enable-force-pf \
...................
```

This will force PF_PACKET over libbpf.
So, libbpf will be pulled as a dependency, but will not be used by `tcpreplay`.